packages/idyll-components/src/header.js

```diff
@@ -89,6 +94,12 @@ Header._idyll = {
       defaultValue: '"#222"',
       description: 'The background of the header. Can pass a color or a url().'
     },
+    {
+      name: 'byLineTemplate',
+      type: 'object',
+      example: "{ prefix: 'Made by', joint: ' -', suffix: '&' }",
```

Comment:
Do we need to include any logic around whitespace? Should we always trim user-provided strings, or just assume extra spaces at the start/end of a string is intentional?

Reply:
@mathisonian That is a good point, currently it adds a space after the prefix and after the joint as well as a space before and after the suffix, which seems like the most usual spacing. But in the example it would result in `Made by A - B - C & D` which seems okay too. We could either make it more strict by trimming or more customizable by letting all white spaces up to the user. WDYT ?

Reply:
I think I'm in favor of making it stricter and always rendering "reasonable spaced" output unless you can think of a use case where that would get in the way.